packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -707,7 +704,7 @@ const ScrollAreaViewport = React.forwardRef(function ScrollAreaViewport(props, f
 
 ScrollAreaViewport.displayName = VIEWPORT_NAME;
 
-/* -------------------------------------------------------------------------------------------------
+/* ---------------------------------------------------  ----------------------------------------------
```

Comment:
Oops, looks like an accidental space was added in the middle here haha

Reply:
😀